frontend/ui/screensaver.lua

```diff
@@ -23,18 +23,41 @@ local _ = require("gettext")
 local Screen = Device.screen
 local T = require("ffi/util").template
 
-local screensaver_provider = {
-    ["jpg"] = true,
-    ["jpeg"] = true,
-    ["png"] = true,
-    ["gif"] = true,
-    ["tif"] = true,
-    ["tiff"] = true,
+-- Migrate old settings
```

Comment:
Some kind of indication might be useful?
```suggestion
-- Migrate old settings from 2021.02 or older.
```